internal/cli/exit.go

```diff
@@ -28,3 +28,17 @@ func Exitf(format string, args ...any) {
 	fmt.Fprintln(os.Stderr, s+fmt.Sprintf(format, args...))
 	os.Exit(1)
 }
+
+// Assert calls Exit if the statement is false.
+func Assert(statement bool, args ...any) {
```

Comment:
Do we need format string for this as well?

Reply:
No, we call `Exit` not `Exitf`.